src/components/pager/css/Pager.css

```diff
@@ -53,6 +53,12 @@
     cursor:default;
 }
 
+/* Pager List Item Conatiner Width */ 
+
+.flc-pager-li-container {
+ width:550px !important; /* FLUID-2797 The prev/next link moves as user clicks previous, Next Button Click Fix */
```

Comment:
I'm not sure this fixed sized width is desirable. As the number of page links changes the gap between them and the next button is quite large. Also, you should try to avoid using `!important`.

Here are a few screen shots as an example.
![screen shot 2016-04-12 at 9 49 36 am](https://cloud.githubusercontent.com/assets/556024/14462285/500dbcc8-0094-11e6-893b-20fda0058fde.png)
![screen shot 2016-04-12 at 9 52 54 am](https://cloud.githubusercontent.com/assets/556024/14462305/5e7964e2-0094-11e6-9142-7b4aaecce7f0.png)